microapps/ArTabletopApp/app/src/main/java/com/arcgismaps/toolkit/artabletopapp/screens/MainScreen.kt

```diff
@@ -61,9 +60,6 @@ fun MainScreen() {
         ArcGISScene().apply {
             operationalLayers.add(arcGISSceneLayer)
             baseSurface = Surface().apply {
```

Comment:
There's no need to create a new Surface, we can just do:
```suggestion
baseSurface.opacity = 0f
```

Reply:
Good catch, updated. Also updated corresponding code snippet in the readme.